### PR TITLE
fix: add locale property

### PR DESCRIPTION
**Pull request by markusahlstrand**
The users were missing a locale property.

Also update the import user script so it writes straight to sql

### CODE REVIEW

**Comment by danielgent** [comment]
Resolved conflicts

**Comment by danielgent** [comment]
The unit tests are pretty unhappy with your changes anyway. :smile:

**Comment by danielgent** [comment]
It's the Node downgrade... That *is* more correct though as it matches our NVM version :thinking: 

I'll look now :eyes:

**Comment by danielgent** [comment]
Pinning node types to 18.13 fixed the unit tests *but* the integration tests aren't happy :smile:

**Comment by danielgent** [comment]
I'll revert any change to the package.json and I'll do this in another PR :wink:

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "better-sqlite3": "^9.2.0",
     "body-parser": "^1.20.2",
     "cookie": "^0.5.0",
+    "csv-writer": "^1.6.0",
```
Comment (danielgent) [inline package.json:56]: where is this used?

**Comment by danielgent** [comment]
The tests are still failing with the downgrade. I'll leave this with you though Markus. I would create separate PRs for each change here

What's the `locale `field for? I don't see this on Auth0 and we're using the tenant's locale to set locales everywhere

**Comment by markusahlstrand** [comment]
Something in here is broken.. Started moving out the pieces to separate PR's..

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -28,6 +28,6 @@ jobs:
       - run: yarn lint
       - run: yarn format:ci-cd
       - run: yarn build
+      - run: yarn type-check
```
Comment (danielgent) [inline .github/workflows/unit-tests.yml:31]: we should type-check straight after building _so_ the CI/CD :x: is more meaningful (it shouldn't show the tests as failing when it's the types that are wrong)

**Comment by danielgent** [comment]
Why would the types be failing? :thinking:   I'll check `main` next. I don't see this PR as having changed anything...

The types are failing on the build output which is from https://github.com/markusahlstrand/tsoa-hono, but we haven't changed that :thinking:

**Comment by danielgent** [comment]
Everything works on `main` :smile:  :sob: 

I'll go onto a few other PRs just to be sure of this :crystal_ball:

File: package.json (hunk 1 above)
Comment (markusahlstrand) [reply package.json:56]: I think we used this in a script to import/export users. So.. should be a dev-dependency or.. maybe a separate package json?

Comment (danielgent) [reply package.json:56]: It's only for us right? everything is tree-shaken before building, only the used imports are loaded.  This isn't actually a Node app so no dependencies are even loaded :smile:

**Comment by danielgent** [comment]
I've recreated this PR based on the diff and the CI/CD passes https://github.com/sesamyab/auth/pull/365

*but* the `yarn.lock` diff is only one line there :smile: 

So with all the additions and deletions this `yarn.lock` has all kinds of unneeded changes which must have made some issues

I'll revert `yarn.lock` to the version on `main` and then push that up here

**Review by danielgent — APPROVED** [review]

**Pull request merged**